docs/processors/standalone/building.mdx

```diff
@@ -289,6 +289,86 @@ func main() {
 
 Check [Compiling the processor](#compiling-the-processor) for what to do next, and how to compile the processor.
 
+## Schemas
+
+Processors have access to the schemas in the used Schema Registry using two utility functions:
+1. `schema.Create` : You can use this utility function to create a new schema and add it to the Schema Registry.
+This function can be called in any of the main processor methods.
+    
+**Example**:
+```go
+func (p *setProcessor) Open(ctx context.Context) error {
+	// Add a new schema to the schema registry before starting to process the records.
```

Comment:
Nitpick: we have a helper for building schemas: https://github.com/ConduitIO/conduit-commons/blob/main/schema/avro/avro_builder.go#L37.

Reply:
@hariso it took me some time to get that i needed to import two different `avro` packages to create the schema byte array (you need `"github.com/conduitio/conduit-commons/schema/avro"` and `avro2 "github.com/hamba/avro/v2"` to create something that looks like this:

> 	avro.NewBuilder("record", "records").AddField("int_field", avro2.NewPrimitiveSchema(avro2.Int, nil), avro2.WithDefault(100)).MarshalJSON()

I think this is a bit more complicated for the user to understand here and not the right place to document this method, WDYT? do we have it documented somewhere else in the docs?
I believe for the sake of the example here, the pure string is easier to understand, but we could link to the builder too.

Reply:
You're right, and good that you're calling it out, we should rename the package then :+1: